diff -crB SFML/src/SFML/Makefile SFMLnew/src/SFML/Makefile
*** SFML/src/SFML/Makefile	2010-10-02 14:42:21.003525297 +0200
--- SFMLnew/src/SFML/Makefile	2010-10-02 14:44:51.168026035 +0200
***************
*** 1,5 ****
  export DEBUGBUILD = no
! export STATIC     = no
  
  ifeq ($(DEBUGBUILD), yes)
      DEBUGFLAGS = -g -DDEBUG
--- 1,5 ----
  export DEBUGBUILD = no
! export STATIC     = yes
  
  ifeq ($(DEBUGBUILD), yes)
      DEBUGFLAGS = -g -DDEBUG
***************
*** 20,26 ****
  export CFLAGS     = -W -Wall -pedantic -I../../../include -I../../ $(DEBUGFLAGS) $(BUILDFLAGS)
  export CFLAGSEXT  = -I../../../include -I../.. $(DEBUGFLAGS) $(BUILDFLAGS)
  export LDFLAGS    = $(LINKFLAGS)
! export LIBPATH    = ../../../lib
  export VERSION    = 1.7
  export CP         = cp
  export LN         = ln
--- 20,26 ----
  export CFLAGS     = -W -Wall -pedantic -I../../../include -I../../ $(DEBUGFLAGS) $(BUILDFLAGS)
  export CFLAGSEXT  = -I../../../include -I../.. $(DEBUGFLAGS) $(BUILDFLAGS)
  export LDFLAGS    = $(LINKFLAGS)
! export LIBPATH    = ../../../../lib/SFML
  export VERSION    = 1.7
  export CP         = cp
  export LN         = ln
